api/client/connection.go

```diff
@@ -33,9 +33,10 @@ var (
 	ErrPoolExists = errors.New("pool exists")
 )
 
-type ReadCloser struct {
+type Response struct {
 	io.ReadCloser
```

Comment:
`Response` is definitely a better name for this! Similar types call their `io.ReadCloser` field `Body`. We might want to do the same.

Reply:
Ha, yes the ReadCloser name was starting to drive me nuts! Made no sense. Who thought of that name?

Reply:
Yeah I can make this Body.